tests/gemm/test_sm_constraint_gemm.py

```
@@ -155,6 +155,76 @@ def test_sm_constraint_gemm(M, N, K, alpha, beta, num_sms, dtype, EPILOGUE_SUBTI
+    # single read turns the whole result into NaN.
+    if kernel == "descriptor_persistent" and dtype == torch.float32:
+        pytest.skip("descriptor persistent does not support float32")
+
+    M = N = K = 256
+    a = torch.randn((M, K), device="cuda", dtype=torch.float16).to(dtype)
+    b = torch.randn((K, N), device="cuda", dtype=torch.float16).to(dtype)
+
+    if kernel == "naive":
+        run = lambda c: flashinfer.triton.sm_constraint_gemm.gemm(
+            a, b, c=c, alpha=alpha
+        )
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `run` as a `def`

(E731)